File: frontend-ADFD/lib/model/place.dart
class Place {
  final int id;
  final String placeName;
  final String address;
  final String img;

  Place({
    required this.id,
    required this.placeName,
    required this.address,
    required this.img,
  });

  factory Place.fromJson(Map<String, dynamic> json) {
    return Place(
      id: json['id'],
      placeName: json['placeName'],
      address: json['address'],
      img: json['img'],
    );
  }
}

File: backend-ADFD/src/main/java/com/example/backendadfd/model/Place.java
package com.example.backendadfd.model;


import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;
import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
@Entity
@Table(name = "place")
public class Place {
    @Id
    private long id;

    @Column(name = "place_name")
    private String placeName;
    @Column(name = "address")
    private String address;
    @Column(name = "img")
    private String img;
}
